contracts/package.json

```diff
@@ -53,7 +53,8 @@
     "solidity-coverage": "^0.8.2",
     "ts-node": "^10.9.1",
     "typechain": "^8.3.2",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@noble/hashes": "1.3.2"
```

Comment:
Note that v1.3.2 is used explicitly here, as that's what ethers depends on.

I guess we could use ^1.3.2 as latest viem uses 1.5.0